### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by kbardi**
# PR for Release v.2.2.1

## 📆 Release Date
The morning of **Tuesday, 28 January 2020** AEST Time

## 📌 Changes
- Letter spacing in featurefont
- Luna font
- Button styles improvements
- Title lockup updates

### Fixed
- Letter spacing in featurefont
- Luna font
- Button styles improvements
- Title lockup updates

## 🚥 Verify/Complete Before Release
- [ ] Check everything works well on mobile and responsive sizes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,11 +85,11 @@ After you created the component structure folder, you wrote unit test, you wrote
 Nice, so you have already created a new component :+1:, that is great!
 It's time to check if everything works as expected.
 
-*If you want to **locally test a component from within your project**. See [How to locally test blueprint library in your project](#how-to-locally-test-blueprint-library-in-your-project) Otherwise, read on to test your built component in blueprint.*
+_If you want to **locally test a component from within your project**. See [How to locally test blueprint library in your project](#how-to-locally-test-blueprint-library-in-your-project) Otherwise, read on to test your built component in blueprint._
```
Comment (bronz3beard) [inline README.md]: I think `How to locally test blueprint library in your project` needs to go here right? [how-to-locally-test-blueprint-components-in-your-project](https://github.com/aimementoring/blueprint#how-to-locally-test-blueprint-components-in-your-project)

File: README.md [hunk 2]
```diff
@@ -70,13 +70,13 @@ Now you are ready to use them :)!
 We have a folder called **commands** where I added a command to create a new component with all folder structure. The way you can use it is running in the terminal inside commands folder `sh createComponent componentName` replacing componentName by your component. You have to write the component name in _camelCase_ notation.
```
Comment (bronz3beard) [inline README.md]: After following the instructions below, I felt like they are useful for someone already familiar with the project.
I would like to propose an alternative approach, that can be integrated or replace existing instructions, or we can leave it as maybe its just the way I work haha.

[Blueprint Intro Doc](https://docs.google.com/document/d/1Up3OSWbhMaFF3n1QlsxxSTznCX4k7x4HTtewxOF7PqA/edit?usp=sharing)

File: README.md (hunk 1 above)
Comment (lulen11) [reply README.md]: I don't understand @bronz3beard 😬

File: README.md (hunk 2 above)
Comment (lulen11) [reply README.md]: Not a bad suggestion @bronz3beard. Always so good to improve on these instructions. Would you be able to open a PR with your suggested readme edits? I've been reading your G Doc and love it but not sure where you want to replace the content. Thank you so much! Such a help!

File: README.md (hunk 1 above)
Comment (bronz3beard) [reply README.md]: Hey Lara,

_"If you want to locally test a component from within your project. See_ `(this hash link doesn't work -> [How to locally test blueprint library in your project](how-to-locally-test-blueprint-library-in-your-project))' _Otherwise, read on to test your built component in blueprint."_

The hash link above didn't take me anywhere? I found the info I needed in the [commands list](https://github.com/aimementoring/blueprint#npm-commands) and this [link](https://github.com/aimementoring/blueprint#how-to-locally-test-blueprint-components-in-your-project) to get the project started locally.

I think the hash link above should link to [here](https://github.com/aimementoring/blueprint#how-to-locally-test-blueprint-components-in-your-project)

Comment (lulen11) [reply README.md]: oh! I get you! my mistake. I'll fix thaat up. thank you!

File: src/lib/components/button/button.module.scss [hunk 3]
```diff
@@ -62,16 +64,5 @@
...
   font-size: 0.8rem;
   display: block;
-  margin-top: 1rem;
-}
-
-.linkButton {
-  padding: 1rem 2.5rem;
-  
-  @media screen and (min-width: 52em) {
-    padding: 0.7rem 2.2rem;
```
Comment (CarolinaPd) [inline src/lib/components/button/button.module.scss]: Sorry just a detail and question really (to know how it works), some margins/paddings use rem and others em, I thought rem was just for fonts and em for the rest, is that correct? Apart from that everything seems okey to me!

File: README.md (hunk 2 above)
Comment (kbardi) [reply README.md]: Same as Lara mentioned, I think it is good, and it is always welcome some improvements in the docs, so if it is possible, could you create a PR with your updates please? thanks @bronz3beard

File: src/lib/components/button/button.module.scss (hunk 3 above)
Comment (kbardi) [reply src/lib/components/button/button.module.scss]: It is a good question, I have the same question...I didn't update styles here, I think rem should be used for fonts, but @lulen11 is the right person to answer that, haha

**Pull request merged**